demangle/src/lib.rs

```diff
@@ -1,26 +1,27 @@
-//! Provides demangling support.
+//! Demangling support for various languages and compilers.
 //!
-//! Currently supported languages:
+//! Currently supported languages are:
 //!
-//! * C++ (without windows)
-//! * Rust
-//! * Swift
-//! * ObjC (only symbol detection)
+//! - C++ (GCC-style compilers and MSVC)
+//! - Rust (both `legacy` and `v0`)
+//! - Swift (up to Swift 5.2)
+//! - ObjC (only symbol detection)
 //!
-//! As the demangling schemes for different languages are different, the
-//! feature set is also inconsistent.  In particular Rust for instance has
-//! no overloading so argument types are generally not expected to be
-//! encoded into the function name whereas they are in Swift and C++.
+//! As the demangling schemes for the languages are different, the supported demangling features are
```

Comment:
how did you manage to format the markdown here? ;-)

Reply:
I'm using an extension called "Rewrap" in VSCode.